webui/react/src/components/CreateGroupModal.tsx

```diff
@@ -214,15 +201,11 @@ const CreateGroupModalComponent: React.FC<Props> = ({ onClose, users, group }: P
                   optionFilterProp="children"
                   placeholder={'Add Roles'}
                   showSearch>
-                  {Loadable.isLoaded(roles) && (
-                    <>
-                      {roles.data.map((r) => (
-                        <Select.Option key={r.id} value={r.id}>
-                          {r.name}
-                        </Select.Option>
-                      ))}
-                    </>
-                  )}
+                  {roles.getOrElse([]).map((r) => (
```

Comment:
i think this part is missing.

The ticket says

> - Sort: Highest access to lowest access
>   - Cluster Admin
>   - Workspace Admin
>   - Editor
>   - Workspace Creator
>   - Viewer
>

Reply:
It's not the order originally in the ticket, but I just talked to Alex and we agreed on sorting by role id so I'm making that change now.